cmd/single-ocsp/main.go

```diff
@@ -26,7 +26,7 @@ type PKCS11Config struct {
 	TokenLabel      string
 	PrivateKeyLabel string
 	PIN             string
-	SlotID          float64
+	SlotID          uint
```

Comment:
In general, using uint types to represent "I swear this number will never be negative" is bad practice. Here's one reference, there are many more for the finding: http://www.soundsoftware.ac.uk/c-pitfall-unsigned. It's better to use ints and do bounds checking if that's what you want.

The API we're calling takes uints, but that doesn't mean we need to plumb that type all the way through. My choice to use uint, and cast inside pkcs11key.New, was a conscious one.